src/DB/exerciseData.json

```diff
@@ -250,15 +250,13 @@
     "image": "https://www.inspireusafoundation.org/wp-content/uploads/2021/10/t-bar-row-muscles.gif"
   }
```

Comment:
You missed a comma here buddy. You could validate your json using a json validator. Either VS extension or https://jsonlint.com. That's why you were getting those deployment errors.